### DISCUSSION TITLE
V2 production ready

### DISCUSSION THREAD

**Discussion by Melancholism** [KevinVandy/material-react-table#755] [Q&A] [answered]
Hey there. Great job and an incredible library. Thank you very much.
Is it recommended to use **V2** in _production_ in this state?

**Comment by KevinVandy** [comment, marked as answer]
I had originally planned to release it as stable yesterday, but decided to wait until the weekend. It's pretty much there with some more of the last parts of the docs being updated.

**Reply by Melancholism** [reply]
Appreciate your quick response.

**Comment by suff42** [comment]
I am using it in production but I'm watching closely all the changes to make sure we are up to date
